clients/unshaded/src/main/java/tachyon/client/ClientOptions.java

```diff
@@ -35,8 +34,8 @@
     private TachyonStorageType mTachyonStorageType;
     /** How this operation should interact with the under storage */
     private UnderStorageType mUnderStorageType;
-    /** Worker location to execute this operation, if not possible, the operation will fail */
-    private NetAddress mLocation;
+    /** Worker location to write data, if not possible, the operation will fail */
```

Comment:
Good point, maybe client options is not the best way of doing it, and it should just be the location policy.